ClientApp/viking-vault/src/components/AccountPage.tsx

```diff
@@ -1,37 +1,74 @@
 import React from 'react'
 import './styles.css'
-import account from './images/card.png'
+import inexistentCard from './images/card.png'
+import genericCard from './images/GENERICcard-01.png'
 import {constants} from "../ConstantVariables";
 import { Redirect } from 'react-router-dom';
 import { TransactionList } from '../UserDashboard/TransactionList';
 
-const url = constants.baseUrl+"Accounts";
+const url = constants.baseUrl+"Accounts/";
+
+interface IAccountBalance{
+    ronBalance: number;
+    eurBalance: number;
+    usdBalance: number;
+    yenBalance: number;
+}
 
 interface IAccountState{
-    balance: number
-    card: any
-    transactions: []
+    firstName: string
+    lastName: string
+    cardNumber: string
+    expirationDate: string
+    accountsBalance: IAccountBalance
+    totalBalance: number
     isPresent: boolean
     redirect: boolean
 } 
 
 class AccountPage extends React.Component<any, IAccountState>{
+    
     constructor(props:any){
         super(props);
+        
+        let defaultBalances: IAccountBalance = {
+            ronBalance: 0,
+            eurBalance: 0,
+            usdBalance: 0,
+            yenBalance: 0
+    
+        };
 
         this.state={
-            balance: 0,
-            card: {},
-            transactions: [],
-            isPresent: false,
+            firstName: "",
+            lastName: "",
+            cardNumber: "",
+            expirationDate: "",
+            accountsBalance: defaultBalances,
```

Comment:
Why do we but defaultBalances in accountsBallance? These names aren`t really clear.